### PR TITLE
Create Quazar Conundrum.yaml

### PR DESCRIPTION
**Pull request by AurthorParadoxQ**
Submitting this manifest for review and hopeful addition to the central plugin repository

### CODE REVIEW

File: manifests/Quazar Conundrum.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/tree/main
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galxy spanning threat, changing hegemony and new factions rising to power.
+url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/releases/tag/Major_ReleaseV015
```
Comment (MCOfficer) [inline manifests/Quazar Conundrum.yaml:8]: this needs to be a direct link to a zip file, such as
```suggestion
url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/Major_ReleaseV015.zip
```

File: manifests/Quazar Conundrum.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/tree/main
+license: CC-BY-SA-4.0
+version: V0.1.5
```
Comment (MCOfficer) [inline manifests/Quazar Conundrum.yaml:5]: If you want to use autoupdate, this will need to be the same as your tag name (currently `Major_ReleaseV015`)

**Comment by AurthorParadoxQ** [comment]
ok i missed some peice of the syntax and now i need to edit it

**Review by MCOfficer — CHANGES_REQUESTED** [review]

File: manifests/Quazar Conundrum.yaml (hunk 2 above)
Comment (AurthorParadoxQ) [reply manifests/Quazar Conundrum.yaml:5]: oh god do i need to resubmit the PR? because i cant figure out how to edit the YAML from here

Comment (AurthorParadoxQ) [reply manifests/Quazar Conundrum.yaml:5]: alright this section is fixed, it should link directly to the release with version

**Comment by AurthorParadoxQ** [comment]
agian? i dont understand why it keeps failing

**Comment by AurthorParadoxQ** [comment]
oh for the love of...the auto updater is still invailed

File: manifests/Quazar Conundrum.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/releases/tag/Major_ReleaseV015
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galxy spanning threat, changing hegemony and new factions rising to power.
+url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/Major_ReleaseV015.zip
+autoupdate:
+	type: tag
+	url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/$version.zip
```
Comment (MCOfficer) [inline manifests/Quazar Conundrum.yaml:11]:
```suggestion
  type: tag
  url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum
```
this should fix your syntax errors - it's just whitespaces.

I was typing out a long essay about autoupdate, but you found the solution on your own. this is correct, but be aware that the first autoupdate run will replace your chosen version with `Major_ReleaseV015`. You can always name your tags differently to change that.

File: manifests/Quazar Conundrum.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/releases/tag/Major_ReleaseV015
```
Comment (MCOfficer) [inline manifests/Quazar Conundrum.yaml:3]: This should point to your github repo, though
```suggestion
homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/
```

**Review by MCOfficer — CHANGES_REQUESTED** [review]

File: manifests/Quazar Conundrum.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galxy spanning threat, changing hegemony and new factions rising to power.
```
Comment (TheGiraffe3) [inline manifests/Quazar Conundrum.yaml:7]:
```suggestion
description: The Quazar Conundrum is a massive century spanning story involving a galaxy-spanning threat, changing hegemony and new factions rising to power.
```
I would also recommend you note that it's still a work in progress, so maybe:
```suggestion
description: The WIP Quazar Conundrum will be a massive century spanning story involving a galaxy-spanning threat, changing hegemony and new factions rising to power.
```

**Comment by AurthorParadoxQ** [comment]
alright pull should be ready to merge at your lesure

File: manifests/Quazar Conundrum.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galaxy-spanning threat, changing hegemony and new factions rising to power.
+url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/Major_ReleaseV015.zip
+autoupdate:
+  type: tag
+  url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum
```
Comment (MCOfficer) [inline manifests/Quazar Conundrum.yaml:11]:
```suggestion
  url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/$version.zip
```
this one's my bad, i made a suggestion before you got it right, so it applying that reverted it back to the wrong state.

**Comment by MCOfficer** [comment]
Ah right... your release's tag name does not fit into the script's idea of a version number. If you want to continue using autoupdate, I need you to create a new release with a tag name such as `V0.1.5`, `v0.1.5` or `v0.1.6`. And continue using that scheme from now on.

File: manifests/Quazar Conundrum.yaml (hunk 6 above)
Comment (AurthorParadoxQ) [reply manifests/Quazar Conundrum.yaml]: oh great

Comment (AurthorParadoxQ) [reply manifests/Quazar Conundrum.yaml]: so what should i do to fix it? make a new PR?

Comment (AurthorParadoxQ) [reply manifests/Quazar Conundrum.yaml]: oh nver mind, i see you fixed it already

**Comment by MCOfficer** [comment]
I removed the autoupdate from your manifest for the time being.

ping @AurthorParadoxQ

**Comment by AurthorParadoxQ** [comment]
ah, alright ill get that fixed, i didnt see this untill today

**Comment by AurthorParadoxQ** [comment]
i also dont know if there is going to be another release given the current state of my life and the glacial progress of the plugin

**Pull request merged**